nifi-registry-docker/pom.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Thanks for putting this together!

I noticed the Travis build for this PR failed due to the RAT check... my guess is its because this file needs the Apache license header.

You can run -Pcontrib-check when you build locally to ensure the build passes styling and license checks.

Reply:
Good point, updated as requested.